portal/static/js/src/profile.js

```diff
@@ -1599,8 +1599,14 @@ export default (function() {
                             }
                             if (self.isSubStudyTriggersResolved()) return;
                             //initialize datepicker
+                            console.log("timestamp? ", self.subStudyTriggers.date)
```

Comment:
I think this is leftover debugging logging?

If it's still needed, could you make it more obvious what's being logged (see below), otherwise remove it?
```
console.log("datepicker initialized with trigger start date: ", self.subStudyTriggers.date);
```

Reply:
thanks, ivan. it was for debugging. I did just remove it.